src/stores/TwoFactorAuthentificationStore.ts

```diff
@@ -133,5 +134,13 @@ export const useTwoFactorAuthentificationStore: StoreDefinition<
         this.loading = false;
       }
     },
+    async resetToken(personId: string): Promise<void> {
+      this.loading = true;
+      try {
+        await twoFactorApi.privacyIdeaAdministrationControllerResetToken(personId);
+      } finally {
+        this.loading = false;
+      }
+    },
   },
 });
```

Comment:
Statt in der Component die Fehler zu catchen bitte hier machen.

Reply:
done